modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -6987,6 +6987,7 @@ enable-tensorflow-image-auto-tagging-on-this-instance=Enable TensorFlow Image Au
 enable-terms-of-use=Enable Terms of Use
 enable-this-for-resources-that-are-hosted-on-a-different-domain=Enable this for resources that are hosted on a different domain.
 enable-to-retain-facet-selections-across-searches=Enable to retain facet selections across searches. When disabled, facets will be reset when a new keyword is searched.
+enable-tooltip-support-description=Enable automatic ClayUI tooltip design across entire site.
```

Comment:
Do we plan to leave this here for customers who did not migrate to `import` and rely on this being injected?

In that case, setting might need some more explanation, such as: "When disabled, toolitp support must be imported in your JS code".

Reply:
The goal is for customers to be able to remove the custom tooltips we provide OOTB. If they disable, they don't necessarily need to provide their own, as the native browser ones still work.

This just makes it so they can disable one more request on page load if needed.